dxsp/handler/client.py

```diff
@@ -148,7 +148,7 @@ async def replace_instrument(self, instrument):
         if self.mapping is None:
             return instrument
         for item in self.mapping:
-            if item["id"] == instrument and item["enable"] is not False:
```

Comment:
<a href="https://sourcery.ai"><img src="https://sourcery.ai/favicon.svg" height="15" alt="Sourcery logo" /></a>  **issue (bug_risk):** Removal of enable check may introduce unintended behavior.

The removal of the check for item['enable'] not being False could lead to processing items that were explicitly disabled. Please ensure this change aligns with the intended logic and consider any potential side effects.